src/libraries/Common/src/Interop/Windows/Advapi32/Interop.GetSecurityDescriptorLength.cs

```diff
@@ -8,8 +8,7 @@ internal static partial class Interop
 {
     internal static partial class Advapi32
     {
-        [DllImport(Interop.Libraries.Advapi32, EntryPoint = "GetSecurityDescriptorLength", CallingConvention = CallingConvention.Winapi,
-            CharSet = CharSet.Unicode, ExactSpelling = true)]
-        internal static extern /*DWORD*/ uint GetSecurityDescriptorLength(IntPtr byteArray);
+        [GeneratedDllImport(Libraries.Advapi32, EntryPoint = "GetSecurityDescriptorLength", CharSet = CharSet.Unicode, ExactSpelling = true)]
```

Comment:
I guess we are just removing CallingConvention = CallingConvention.Winapi from everywhere as it's default and archaic.